examples/customized-reporter/.sauce/config.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1alpha
+kind: playwright
+sauce:
+  region: us-west-1
+  concurrency: 20 # Controls how many suites are executed at the same time.
+  metadata:
+    tags:
+      - e2e
+      - release team
+      - other tag
+    build: Release $CI_COMMIT_SHORT_SHA
+docker:
+  # Affects how test files are transferred to the docker container when using the docker mode.
+  fileTransfer: copy # Choose between mount|copy.
+playwright:
+  version: 1.22.2 # See https://docs.saucelabs.com/dev/cli/saucectl/#supported-frameworks-and-browsers for a list of supported versions.
```

Comment:
Hm. I suppose this shouldn't be merged until we release the next playwright runner version, since it'll rely on https://github.com/saucelabs/sauce-playwright-runner/pull/159 to actually work. And at that point, this version here would have to be bumped too.

Reply:
make sense.